guides/common/modules/proc_upgrading-smartproxy-server.adoc

```diff
@@ -139,3 +139,19 @@ include::snip_steps-needs-reboot.adoc[]
 ifdef::orcharhino,satellite[]
 . Optional: If you use custom repositories, ensure that you enable these custom repositories after the upgrade completes.
 endif::[]
+
+.Upgrading {SmartProxyServersTitle} using remote execution
+
+. Create a backup or take a snapshot.
++
+For more information on backups, see {AdministeringDocURL}backing-up-satellite-server-and-capsule-server[Backing Up {ProjectServer} and {SmartProxyServer}] in the _Administering {ProjectName} {ProjectVersionPrevious}_ guide.
+. In the {ProjectWebUI} , navigate to *Monitor* > *Jobs*.
+. Click *Run Job*.
+. From the *Job category* list, select *Maintenance Operations*.
+. From the *Job template* list, select *{SmartProxy} Upgrade Playbook*.
+. In the *Search Query* field, enter the host name of the {SmartProxy}.
+. Ensure that *Apply to 1 host* is displayed in the *Resolves to* field.
+. In the *target_version* field, enter the target version of the {SmartProxy}.
+. In the *whitelist_options* field, enter the options.
+. In the *Type of query* section, click *Static Query*.
+. Select the schedule for the job execution in *Schedule*.
```

Comment:
The "Schedule" and "Query Type" steps should be reversed because the Schedule section is followed by the Query Type.

![Screenshot_20240304_152344](https://github.com/theforeman/foreman-documentation/assets/31725355/7c1a162f-6222-4c0c-b54e-727104a8ae74)